src/mutable_data.rs

```diff
@@ -876,7 +932,7 @@ impl SeqEntryActions {
         self
     }
 
-    /// Delete an entry
+    /// Deletes an entry
```

Comment:
Missing full stop:
```suggestion
    /// Deletes an entry.
```
It's also worth explaining what the `version` parameter means and what it should be equal to.